src/aggregator/aggregator/follower_flush_mgr.go

```diff
@@ -136,7 +136,7 @@ func newFollowerFlushManager(
 	scope := instrumentOpts.MetricsScope()
 	mgr := &followerFlushManager{
 		nowFn:                    nowFn,
-		checkEvery:               opts.CheckEvery(),
+		checkEvery:               time.Second * 5, // opts.CheckEvery(),
```

Comment:
NB: this is a temporary hack to force the timing issue to manifest more reliably, will be reverted